app.rb

```diff
@@ -48,10 +48,12 @@ def get_logger(progname, threshold=nil)
 # Setup Elasticsearch
 # NOTE (CCB): If you want to see all data sent to Elasticsearch (e.g. for debugging purposes), set the tracer argument
 # to the value of a logger.
-# Example: Elascisearch.Client.new(tracer: get_logger('elasticsearch.tracer'))
+# Example: Elasticsearch::Client.new(tracer: get_logger('elasticsearch.tracer'))
+# NOTE: You can also add a logger, but it will log some FATAL warning during index creation.
+# Example: Elasticsearch::Client.new(logger: get_logger('elasticsearch', Logger::WARN))
```

Comment:
See [earlier PR comment](https://github.com/edx/cs_comments_service/pull/229/files#r98747334), as well as this code comment, regarding why I yanked this logging.  Is everyone ok with this?
FYI: @clintonb @dianakhuang @maxrothman